tests/framework/Models/External/tests

```diff
@@ -4,6 +4,23 @@
   input = 'all_methods.xml'
   csv = 'AllMethods/samples_out.csv'
  [../]
+ [./serialize_ext_model]
+  type = 'RavenFramework'
+  input = 'serialize_ext_model_and_use.xml'
+  csv = 'SerializeWrkd/samples_out.csv'
+ [../]
+ [./fmu_ext_model]
+  type = 'RavenFramework'
+  input = 'fmu_ext_model_and_use.xml'
+  output = 'SerializeWrkd/attenuate_pk2.fmu'
+ [../]
+ [./loadFMU]
+  type = 'RavenPython'
+  input = 'testMathUtils.py'
```

Comment:
Hm, this probably should be load_and_run_fmu.py

Reply:
fixed